e2e/playwright/point-click.spec.ts

```diff
@@ -677,3 +677,94 @@ test(`Offset plane point-and-click`, async ({
     await scene.expectPixelColor([74, 74, 74], testPoint, 15)
   })
 })
+
+const loftPointAndClickCases = [
+  { shouldPreselect: true },
+  { shouldPreselect: false },
+]
+loftPointAndClickCases.forEach(({ shouldPreselect }) => {
+  test(`Loft point-and-click (preselected sketches: ${shouldPreselect})`, async ({
+    app,
+    page,
+    scene,
+    editor,
+    toolbar,
+    cmdBar,
+  }) => {
+    const initialCode = `sketch001 = startSketchOn('XZ')
+  |> circle({ center = [0, 0], radius = 30 }, %)
+plane001 = offsetPlane('XZ', 50)
+sketch002 = startSketchOn(plane001)
+  |> circle({ center = [0, 0], radius = 20 }, %)
+`
+    await app.initialise(initialCode)
+
+    // One dumb hardcoded screen pixel value
+    const testPoint = { x: 575, y: 200 }
+    const [clickOnSketch1] = scene.makeMouseHelpers(testPoint.x, testPoint.y)
+    const [clickOnSketch2] = scene.makeMouseHelpers(
+      testPoint.x,
+      testPoint.y + 80
+    )
+    const loftDeclaration = 'loft001 = loft([sketch001, sketch002])'
+
+    await test.step(`Look for the white of the sketch001 shape`, async () => {
+      await scene.expectPixelColor([254, 254, 254], testPoint, 15)
+    })
+
+    async function selectSketches() {
+      await clickOnSketch1()
+      await page.keyboard.down('Shift')
+      await clickOnSketch2()
+      await app.page.waitForTimeout(500)
+      await page.keyboard.up('Shift')
+    }
+
+    if (!shouldPreselect) {
+      await test.step(`Go through the command bar flow without preselected sketches`, async () => {
+        await toolbar.loftButton.click()
+        await cmdBar.expectState({
+          stage: 'arguments',
+          currentArgKey: 'selection',
+          currentArgValue: '',
+          headerArguments: { Selection: '' },
+          highlightedHeaderArg: 'selection',
+          commandName: 'Loft',
+        })
+        await selectSketches()
+        await cmdBar.progressCmdBar()
+        await cmdBar.expectState({
+          stage: 'review',
+          headerArguments: { Selection: '2 faces' },
+          commandName: 'Loft',
+        })
+        await cmdBar.progressCmdBar()
+      })
+    } else {
+      await test.step(`Preselect the two sketches`, async () => {
+        await selectSketches()
+      })
+
+      await test.step(`Go through the command bar flow with preselected sketches`, async () => {
+        await toolbar.loftButton.click()
+        await cmdBar.progressCmdBar()
```

Comment:
After discussing elsewhere, we're going to leave this (and the skip logic) until we can put in a small refactor PR to let validation be configurable as a callback on the selection. Then we'll come back, use that callback so that it only skips if there are `>2 sketches` in the selection